src/web/components/ArticleHeadline.tsx

```diff
@@ -183,7 +186,28 @@ const renderHeadline = ({
         colour?: string;
     };
 }) => {
+    if (display === 'immersive') {
```

Comment:
so `designType` has capitalisation ex: Immersive
and display lowercase

Reply:
Indeed, `pillar` is lower so I followed that. Either way we're inconsistent. We can look to refactor capitalisation or even move over to typescript's const strings later